utils/config/daos_server.yml

```diff
@@ -290,7 +290,7 @@
 #  # For best performance, it is necessary that the fabric_iface of this engine
 #  # resides on the same NUMA node as the first_core.
 #  #
-#  # Optional parameter; set either this option or pinned_numa_node but not both.
+#  # Optional parameter; set either this option non-zero or pinned_numa_node but not both.
```

Comment:
Agreed. I think the "core offset" parameter doesn't make sense to NUMA mode, and I've already changed io engine in multiprovider feature branch to ignore the 'core offset' parameter for NUMA mode.